internal/pkg/agent/install/uninstall.go

```diff
@@ -36,7 +36,14 @@ func Uninstall(cfgFile, topPath, uninstallToken string) error {
 	if err != nil {
 		return err
 	}
-	status, _ := svc.Status()
+
+	status, err := svc.Status()
+	if err != nil {
```

Comment:
This converts this into a fatal error for the install process? Is this actually necessary?

We have a few open issues where the agent doesn't respond to service manager events properly, so I'm not sure this is the right choice.

Can we log this without aborting the uninstall process?

Reply:
Ah we won't attempt to stop the service if we can't detect that it is running, so not stopping an unresponsive service probably isn't correct either.

Reply:
what if we make the force flag an argument to `Uninstall`?  That way we can optionally ignore the error messages (but print to stderr) and try to continue.  But it most cases, if Status() returns an error, than Stop() is going to error as well.

As a side note.  Stop() can actually return without an error but the service isn't in the Stopped state.
https://github.com/kardianos/service/blob/9832e01049dd49bb66dd8fc447c72eefba7fc2cd/service_windows.go#L499-L525

If the service doesn't report status.Stopped within the `WaitToKillServiceTimeout` time (default 5 sec), then Stop just returns with a nil error.

Reply:
Force flag works for me, would like to keep a way to do a best effort uninstall given the amount of bugs in Windows service for the agent right now.

Reply:
We are probably also mixing the kardianos handling with the service handlers from Beats https://github.com/elastic/elastic-agent-libs/blob/main/service/service_windows.go and I don't think that is helping anything.